src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -30,14 +29,15 @@ public Car create(Car car) {
             if (resultSet.next()) {
                 car.setId(resultSet.getObject(1, Long.class));
             }
+            addDriverToCar(car);
```

Comment:
it would be better to open new connection after current connection will be closed, so lets move `addDriverToCar(car);` below try-catch block